### DISCUSSION TITLE
Make it sense to introduce further optimiser?

### DISCUSSION THREAD

**Discussion by Anselmoo** [keras-team/autokeras#1734] [Ideas] [answered]
@haifeng-jin would it make sense to implement further [KERAS Optimizer](https://keras.io/api/optimizers/) or just over-complicated the optimization process?

https://github.com/keras-team/autokeras/blob/424c0cc29dc3837f18ca75d927851674e6f40c30/autokeras/graph.py#L292-L322

I assume this could be added via new `elif`-conditions?

thx for feedback

**Comment by haifeng-jin** [comment, marked as answer]
We may add some common ones like in https://github.com/keras-team/autokeras/pull/1736.
However, if the user have special needs, we should recommend them to use KerasTuner.
